src/sharing/components/recipient.styl

```diff
@@ -68,4 +68,5 @@
 
 .pho-action-unshare
     color pomegranate !important
+    text-decoration none !important
```

Comment:
Why do we need the `!important` here?

Reply:
Because of what I think are $@# global link styles I couldn't override...

Reply:
That's a bit worrysome. Do you know where these global styles are defined?

Reply:
I believe it's because of this: https://github.com/cozy/cozy-drive/blob/master/src/sharing/share.styl#L11